rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc

```diff
@@ -2,8 +2,51 @@
 
 If a bot has status *OK*, but seems not to work properly, download its log files or restart the bot.
 
+If the bot has status *SUSPENDED*, have a look at the execution plans.
+
 If a bot has status *STOPPED*, *OVERDUE*, or *UNKNOWN*, an IT administrator needs to log on to the computer the bot is installed on and needs to troubleshoot it there.
 
+== View the Planned Process Executions
+
+If you are not sure why a bot is not processing, have a look at the execution plans:
+
+. In the module *Bot Management*, open the view *Bot Execution Plans*.
+. In the table click the *Execution timetable* icon image:executiontimetable-icon.png[calendar symbol,1.5%,1.5%] in the row of the bot you want to check.
+
+The execution time table shows you the effective runtime of all configuration assigned to the bot.
+
+. Click the *Configure Execution Timetable* icon image:configure-icon.png[wrench symbol,1.5%,1.5%] to change the time interval of the table and to select the shown configurations.
+. Click *OK*.
+. In the *Execution Timetable*, click on a colored bar in the row of the configuration you want to check to view the composition of the effective runtime.
+
+A configuration is executed when the bot's service time overlaps with the schedule of the configuration. If the process uses applications, it is not executed during the downtimes of an application.
+
+All times are displayed in your local time. Times from bots and applications living in other time zones are converted to your local time. Hover over the bars to see which service times, downtimes and schedules are contained in the execution times.
+
+=== Example of the Effective Runtime Composition
+
+image::rpa_botmanagement_botexecutionplans_example_effectiveschedulecomposition.png[Overview of how the effective runtime is composed from schedules, service times and downtimes]
+
+The example shows the planned process executions from the view of a user who lives in Rome (GMT+01:00). All shown times are converted to the user's time zone.
+
+The shown configuration uses a schedule which is repeated daily from 12:30 PM to 20:30 PM regardless of time zones.
+
+The bot is installed on a computer using UTC timezone. It has an assigned service time defined in Japanese time (GMT+09:00).
+
+The process uses an application installed on a computer in Buenos Aires (GMT-03:00). There was a power outage during which the application was down from 10:00 AM till 10:30 AM.
+
+When schedules and service times overlap and no used applications is down, the bot executes the process.
+
+In this example, a downtime takes place in the middle of the scheduled time. The execution is interrupted during that time. Because the service time ends before the scheduled run time, the effective runtime is shortened, too.
```

Comment:
```suggestion
In this example, a downtime takes place in the middle of the scheduled time. The execution is interrupted during that time. Because the service time ends before the scheduled run time, the effective runtime is shortened.
```